src/runtime/components/forms/SelectMenu.vue

```diff
@@ -221,6 +221,14 @@ export default defineComponent({
       type: Boolean,
       default: false
     },
+    clearable: {
+      type: Boolean,
+      default: false
+    },
+    clearableIcon: {
+      type: String,
+      default: 'i-heroicons-x-mark-20-solid'
```

Comment:
This should default from the config:
```suggestion
      default: () => configMenu.default.clearableIcon
```